docs/guide/framework.rst

```diff
@@ -0,0 +1,51 @@
+Framework Client: one client for all services
+=============================================
+
+The Dynamic Annotation Framework consists of a number of different
+services, each with a specific set of tasks that it can perform through
+REST endpoints. This module is designed to ease programmatic interaction
+with all of the various endpoints. Going forward, we also will be
+increasingly using authentication tokens for programmatic access to most
+if not all of the services. In order to collect a given server, dataset
+name, and user token together into a coherent package that can be used
+on multiple endpoints, we will use a FrameworkClient that can build
+appropriately configured clients for each of the specific services. Each of the individual services has their own specific documentation as well.
+
+Initializing a FrameworkClient
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Assuming that the services are on ``www.dynamicannotationframework.com``
```

Comment:
maybe we should have a configuration file? or a set of enviornment variables for configuration?